quickstart/aws/tutorial-eks.md

```diff
@@ -45,17 +45,17 @@ In this tutorial, we'll launch a new Managed Kubernetes cluster in Elastic Conta
 
     ```typescript
     import * as pulumi from "@pulumi/pulumi";
-    import * as awsinfra from "@pulumi/aws-infra";
+    import * as awsx from "@pulumi/awsx";
     import * as eks from "@pulumi/eks";
     import * as k8s from "@pulumi/kubernetes";
 
     const name = "helloworld";
 
     // Create an EKS cluster with non-default configuration
-    const vpc = new awsinfra.Network("vpc", { usePrivateSubnets: false });
+    const vpc = new awsx.ec2.Vpc("vpc");
     const cluster = new eks.Cluster(name, {
-        vpcId: vpc.vpcId,
-        subnetIds: vpc.subnetIds,
+        vpcId: vpc.id,
+        subnetIds: vpc.publicSubnetIds,
```

Comment:
Is this right?  We are changing the example from "public" to "public+private", but then passing the "public" subnets to the cluster, which won't work, right?

Reply:
Why woudl the public subnets not work?  Note: i will change this to be fully public.  however, the presence of private subnets shoudln't affect your ability to use the public ones, right?

Reply:
I am not exactly sure that it doesn’t work (though I suspect it might not), but it is certainly not common to have private sinners but then run compute in the public ones. The typical (only?) intent of public+private subnet arrangement is to have compute on private and ingress/egress through public.